airflow/example_dags/example_kubernetes_executor.py

```diff
@@ -16,88 +16,214 @@
 # specific language governing permissions and limitations
 # under the License.
 """
-This is an example dag for using the Kubernetes Executor.
+This is an example dag for using a Kubernetes Executor Configuration.
 """
+import logging
 import os
 from datetime import datetime
 
 from airflow import DAG
+from airflow.configuration import conf
 from airflow.decorators import task
 from airflow.example_dags.libs.helper import print_stuff
 
+log = logging.getLogger(__name__)
+
+worker_container_repository = conf.get('kubernetes', 'worker_container_repository')
+worker_container_tag = conf.get('kubernetes', 'worker_container_tag')
+
+try:
+    from kubernetes.client import models as k8s
+except ImportError as e:
+    log.warning("Could not import DAGs in example_kubernetes_executor.py: %s", str(e))
+    log.warning("Install kubernetes dependencies with: pip install apache-airflow['cncf.kubernetes']")
+
+
 with DAG(
     dag_id='example_kubernetes_executor',
     schedule_interval=None,
     start_date=datetime(2021, 1, 1),
     catchup=False,
-    tags=['example', 'example2'],
+    tags=['example3'],
```

Comment:
Not particularly related but what do these tags mean in the first place?

Reply:
Just to group dags so one can easily filter the dags in the UI based on the tag. This here is only used for demonstration purpose

Reply:
So… this particular change does not really mean anything?

Reply:
The dag where it was defined is `example_kubernetes_executor_config.py`, I worked on that file and later renamed it to `example_kubernetes_executor.py`. The original file `example_kubernetes_executor.py` was deleted so it appear now as if I worked on it.